.github/workflows/release-airbyte.yml

```diff
@@ -0,0 +1,67 @@
+name: Release Airbyte
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  release-airbyte:
+    if: startsWith(github.event.head_commit.message, 'Bump Airbyte version')
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+      pull-requests: read
+    steps:
+      - name: Fetch Version Bump PR Body
+        id: fetch_pr_body
+        env:
+          COMMIT_ID: ${{ github.event.head_commit.id }}
+        shell: bash
+        run: |-
+          set -x
+          PR=$(curl \
+          -H "Accept: application/vnd.github.v3+json" \
+          -H "Authorization: Bearer ${{ secrets.GITHUB_TOKEN }}" \
+          https://api.github.com/repos/${{ github.repository }}/commits/$COMMIT_ID/pulls)
```

Comment:
confirming my understanding: because this is searching for a specific commit id, this should be fine even if we have multiple release PRs open?

Reply:
Yep, I believe so!